stage_test.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Ideally we should move these k8s YAMLs to `config/` and split them up into separate files, utilizing Kustomize. I can help you with that later on.